__tests__/app.test.js

```diff
@@ -62,6 +62,14 @@ describe("GET /api/articles/:article_id", () => {
     return request(app)
       .get("/api/articles/15")
       .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("Not found");
+      });
+  });
+  test("Status: 404, handles error when Id is invalid", () => {
```

Comment:
You've asked for /notAnId

This is a not a valid id (its not a number)

This is a bad request - 400